armi/physics/fuelCycle/fuelHandlers.py

```diff
@@ -762,6 +762,20 @@ def _transferStationaryBlocks(self, assembly1, assembly2):
                     assembly1, a1StationaryBlocks, assembly2, a2StationaryBlocks
                 )
             )
+        if a1StationaryBlocks[-1][0].p.ztop != a2StationaryBlocks[-1][0].p.ztop:
+            runLog.warning(
+                """Difference in top elevation of stationary blocks 
+                 between {} (Stationary Blocks: {}, Elevation at top of stationary blocks {}) 
+                 and {} (Stationary Blocks: {}, Elevation at top of stationary blocks {}))""".format(
+                    assembly1,
+                    a1StationaryBlocks,
+                    a1StationaryBlocks[-1][0].p.ztop,
+                    assembly2,
+                    a2StationaryBlocks,
+                    a2StationaryBlocks[-1][0].p.ztop,
+                )
+            )
+            return
```

Comment:
probably delete the return statement, and write a test to cover this method if we dont have it.

Reply:
Removed!